scripts/import_backends.py

```diff
@@ -227,7 +227,7 @@ def description_text(self, ticket: Ticket) -> str:
 
         if ticket.dependencies:
             text += self._section("Dependencies", "\n".join(
-                f" * #{dep} {self.title(dep)}".rstrip()
+                f" - [ ] #{dep} {self.title(dep)}".rstrip()
```

Comment:
If I recall GitHub includes the titles too if we do this -- should we drop our explicit inclusion of the title?